src/script/conversation/ConversationMapper.ts

```diff
@@ -69,7 +69,8 @@ export interface SelfStatusUpdateDatabaseData {
 export type ConversationDatabaseData = ConversationRecord &
   Partial<ConversationBackendData> & {
     accessModes?: CONVERSATION_ACCESS[];
-    accessRole?: CONVERSATION_ACCESS_ROLE;
+    //CONVERSATION_ACCESS_ROLE for api <= v2, ACCESS_ROLE_V2[] since api v3
+    accessRole?: CONVERSATION_ACCESS_ROLE | ACCESS_ROLE_V2[];
```

Comment:
I think it can like that:
```suggestion
    accessRole?: CONVERSATION_ACCESS_ROLE | ACCESS_ROLE[];
```